src/main.js

```diff
@@ -6,10 +6,15 @@ import router from '@/router';
 import i18n from '@/i18n';
 import { initCuttleGlobals } from '_/utils/config-utils';
 import App from '@/App.vue';
+import { createHead } from '@unhead/vue';
```

Comment:
Please move this import up higher, so it goes with the other external imports. It can go right under i18n

Reply:
Seems like a follow up issue to add sort imports as an eslint rule :) https://eslint.org/docs/latest/rules/sort-imports

Reply:
Hey @itsalaidbacklife I have updated this import as well to be under the i18n import. Here is the [link](https://github.com/cuttle-cards/cuttle/pull/1145/commits/86f6bc115aa620ac281bc9f6e9125a3f78d3efb5).